address.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
We should not add `address.json` back in.

By default `address.json` comes from the `ocean-contracts` dependency and should not be duplicated here. If the user needs to use custom contract addresses, they can provide their own `address.json` file.

Reply:
agreed, somehow it was added when I pushed files.